packages/typography/src/Link/Link.tsx

```diff
@@ -65,19 +65,30 @@ const Link = InferredPolymorphic<LinkProps, 'span'>(
       }
     }, [rest, currentHostname]);
 
-    let target, rel, icon;
+    let icon;
+
+    const defaultAnchorProps: Pick<
+      JSX.IntrinsicElements['a'],
+      'target' | 'rel'
+    > = {
+      target: undefined,
+      rel: undefined,
+    };
 
-    // Sets defaults for target and rel props when Component is an anchor tag
-    if (Component === 'a') {
+    if ((rest as AnchorLikeProps).target || (rest as AnchorLikeProps).rel) {
+      defaultAnchorProps.target = (rest as AnchorLikeProps).target;
+      defaultAnchorProps.rel = (rest as AnchorLikeProps).rel;
```

Comment:
It's the same number of lines, so take it or leave it, but I find this cleaner:
```
const {target, rel} = rest as AnchorLikeProps
defaultAnchorProps = {target, rel}
```

Reply:
I wanted to leave target and rel in rest so that they override our default settings